### PR TITLE
docs: update go account management page

### PR DESCRIPTION
**Pull request by jmcook1186**
mostly relatively light changes to the account management page, bringing it into line with the rest of the dapp developer docs.

### CODE REVIEW

File: docs/_dapp/native-accounts.md [hunk 1]
```diff
@@ -3,140 +3,107 @@ title: Go Account Management
...
-vulnerabilities.
-
-To support this, `go-ethereum` provides a simple, yet thorough accounts package that gives
-you all the tools to do properly secured account management via encrypted keystores and
-passphrase protected accounts. You can leverage all the security of the `go-ethereum`
-crypto implementation while at the same time running everything in your own application.
+Geth provides a simple, yet thorough accounts package that includes all the tools developers
+need to leverage all the security of Geth's crypto implementation in a Go native application.
+The account management is done client side with all sensitive data held inside the application.
+This gives the user control over access permissions without relying on any third party.
```
Comment (MariusVanDerWijden) [inline docs/_dapp/native-accounts.md:9]: Can you add something here about how `Clef` is the recommended way to manage keys? We plan to phase out native accounts over the long run

File: docs/_dapp/native-accounts.md [hunk 2]
```diff
@@ -3,140 +3,107 @@ title: Go Account Management
...
-provides a higher level of security at the cost of increased computational burden and
-resource consumption:
+## Encrypted keystores
 
- * *standard* needs 256MB memory and 1 second processing on a modern CPU to access a key
- * *light* needs 4MB memory and 100 millisecond processing on a modern CPU to access a key
+Access keys to Ethereum accounts should never be stored in plain-text. Instead, they should be 
+stored encrypted so that even if the mobile device is accessed by a malicious third party the 
+keys are still hidden under an additional layer of security. Geth provides a keystore that enables 
+developers to store keys securely using the [`secp256k1`][secp256k1] elliptic curve,
```
Comment (MariusVanDerWijden) [inline docs/_dapp/native-accounts.md:20]: The keystore uses Scrypt to store the secp256k1 keys. So this phrasing is a bit off.

**Review by MariusVanDerWijden — APPROVED** [review]
Looks good to me, just two small nitpicks

File: docs/_dapp/native-accounts.md (hunk 1 above)
Comment (jmcook1186) [reply docs/_dapp/native-accounts.md:9]: Sure - done!

**Pull request merged**